doc/ug/dv_methodology.md

```diff
@@ -415,6 +415,28 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
+### X-Propagation (Xprop)
+
+It's mandatory to enable Xprop when running regression for coverage closure.
```

Comment:
Added explanation of xprop
Maybe I miss something, but why change it's -> It is?
Prefer to use Xprop as it's the short name used in VCS and align with the opt name

Reply:
Just a minor thing. `It's` is very similar sounding to `Its` and they are often confused with one another. Both semantically different (former is a contraction of `It is` and the latter has a notion of *possession*.  Better to avoid this confusion.  

https://www.grammarly.com/blog/its-vs-its/

Reply:
`It's` isn't `Its`. I feel `it's` should be allowed and we used it in several places in the repo.

Reply:
Sure, not a big deal.